### PR TITLE
feat(client): module update track for graph page

### LINKED ISSUE #177: [Feature Request] Graph should be updated in real time
## description

`Graph` will not update automatically after loading lazy-load routes with vue-router. After switching back to the `Graph` tab, it will update correctly.

https://github.com/vuejs/devtools-next/assets/50254496/fa2ab78b-78f4-4099-ab4e-fc68eb462093

## repro

URL: <https://stackblitz.com/edit/vitejs-vite-nzyucm?file=vite.config.ts>

### PR DESCRIPTION
**Pull request by LoTwT**
## description

This PR is intended to close #177.

### CODE REVIEW

**Comment by LoTwT** [comment]
Hi, @webfansplz.

This is a draft PR including the codes that may cause RPC communications timeout.

Some other problems need to be resolved now. It's just a draft PR now to communicate issues.

It now completes a minimal implementation because `vite-plugin-inspect` hasn't released a new version yet.

This PR uses `server.ws` to send a custom event when some other URLs are requested which means some new resource will be loaded. In the client, it uses a pre-defined vite hot context to listen to the custom event and do something. When using `bridgeRpc` in the vite hot context, the RPC communications will always be timeout. 

steps to start:

- nr dev
- nr play
- check the console in the terminal ( server side )
- open the chrome devtool in the browser ( client side )
- click `Toggle Langs` to mock a new request for resources
- wait and see the console on the client side

**Comment by LoTwT** [comment]
Hi, @webfansplz.

This PR is ready to review now.

Your codes on the RPC communications are cleverly designed. I think they have resolved the scopes referred to in #177. Thank you! ♥️

For the better design of RPC communications, I will later create a new issue.

File: packages/client/src/main.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ import WaitForConnection from '~/components/WaitForConnection.vue'
 import 'uno.css'
 import '~/assets/styles/main.css'
 
-async function getViteHotContext() {
+export async function getViteHotContext() {
```
Comment (webfansplz) [inline packages/client/src/main.ts:30]: I think we can revert this export.

Comment (LoTwT) [reply packages/client/src/main.ts]: I have made a new commit. Thank you. ♥️

**Review by webfansplz — APPROVED** [review]

**Pull request merged**